tests/embedded_graphics.rs

```diff
@@ -148,3 +148,26 @@ fn issue_136_row_size_is_multiple_of_4_bytes() {
         "WWWWKWWWW",
     ]);
 }
+
+/// Test for issue #8
+#[test]
+fn issue_8_height_is_negative() {
+    let image_positive: Bmp<Rgb888> =
+        Bmp::from_slice(include_bytes!("./issue_8-positive.bmp")).unwrap();
+    let image_negative: Bmp<Rgb888> =
+        Bmp::from_slice(include_bytes!("./issue_8-negative.bmp")).unwrap();
+
+    let image = Image::new(&image_positive, Point::zero());
+
+    let mut display = MockDisplay::new();
+    image.draw(&mut display).unwrap();
+
+    display.assert_pattern(&["WK", "KK"]);
+
+    let image = Image::new(&image_negative, Point::zero());
+
+    let mut display = MockDisplay::new();
+    image.draw(&mut display).unwrap();
+
+    display.assert_pattern(&["WK", "KK"]);
+}
```

Comment:
This looks good, but I've refactored it a bit. 

```suggestion
/// Test for issue #8
#[test]
fn issue_8_height_is_negative() {
    let image_bottom_up: Bmp<Rgb888> =
        Bmp::from_slice(include_bytes!("./issue_8-positive.bmp")).unwrap();
    let image_top_down: Bmp<Rgb888> =
        Bmp::from_slice(include_bytes!("./issue_8-negative.bmp")).unwrap();

    assert_eq!(image_bottom_up.as_raw().header().image_data_top_down, false);
    assert_eq!(image_top_down.as_raw().header().image_data_top_down, true);

    let image_bottom_up = Image::new(&image_bottom_up, Point::zero());
    let image_top_down = Image::new(&image_top_down, Point::zero());

    let mut bottom_up_display = MockDisplay::new();
    let mut top_down_display = MockDisplay::new();

    image_bottom_up.draw(&mut bottom_up_display).unwrap();
    image_top_down.draw(&mut top_down_display).unwrap();

    bottom_up_display.assert_pattern(&["WK", "KK"]);
    bottom_up_display.assert_eq(&top_down_display);
}
```

I'm assuming the `positive` BMP is in bottom up order, but the file naming is a bit confusing.

The main points are:

- Renaming the variables to make them consistent with the new property
- Using `MockDisplay::assert_eq` to check the output is the same
- Checking the new field in the header that the order is indeed correct

Bonus points if you feel like renaming the BMP files to include the `top-down`/`bottom-up` naming.